CHANGELOG.md

```diff
@@ -11,6 +17,7 @@
 - More logging for Session Replay video info (#5132)
 - Improve session replay frame presentation timing calculations (#5133)
 - Use wider compatible video encoding options for Session Replay (#5134)
+- GA of new session replay view renderer (#5054)
```

Comment:
`m`: Please change the working to V2 and also the PR title.

Reply:
Not sure if I understand correctly, is the updated version ok?